config/puma.rb

```diff
@@ -9,14 +9,32 @@
 threads_count = ENV.fetch('RAILS_MAX_THREADS') { 5 }
 threads threads_count, threads_count
 
-# Specifies the `port` that Puma will listen on to receive requests; default is 3000.
-#
-port        ENV.fetch('PORT') { 3000 }
-
 # Specifies the `environment` that Puma will run in.
 #
 environment ENV.fetch('RAILS_ENV') { 'development' }
 
+app_dir = File.expand_path('..', __FILE__)
```

Comment:
@sameeraappana8 One last tweak.  Change `__FILE__` to `__dir__`.

Then squash the commits into one and let me know it's ready.